src/components/SearchBar/SearchBar.constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEBOUNCE_DELAY = 300;
+export const DEFAULT_PLACEHOLDER = 'Search';
```

Comment:
DEFAULT_PLACEHOLDER is a hardcoded English string. Per project i18n/L10n rules, UI text should be localizable or provided by consumers via props.

**Sugerencia:**
```typescript
Either document this as a fallback and accept placeholder via props (already present), or make this constant come from a locale map or translation helper. At minimum, mark it as a fallback and avoid using it directly in consumer code if localization is required.
```